drivers/growatt/api.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019: Rob, https://github.com/pro-sumer
+// License: GNU GPLv3, https://www.gnu.org/licenses/gpl.txt
+
+'use strict'
```

Comment:
Might be nice to rewrite this one into an ES6 Class, but not that important for now. If you are interested and have the time, you can look at an example in the Omnik API class of this project.

Reply:
Yes, the Omnik class was an excellent example. Done.